src/thor/multimodal.cc

```diff
@@ -367,8 +367,8 @@ bool MultiModalPathAlgorithm::ExpandForward(GraphReader& graphreader,
 
       // Look up the next departure along this edge
       const TransitDeparture* departure =
-          tile->GetNextDeparture(directededge->lineid(), offset_time.local_time, day_, dow_,
-                                 date_before_tile_, tc->wheelchair(), tc->bicycle());
+          tile->GetNextDeparture(directededge->lineid(), offset_time.local_time % kSecondsPerDay,
```

Comment:
we need the seconds in the week. again, gotta check this holds true for the weirdo sunday-first-day-of-week stuff